IssueTracker/IssueTrackerAPI/Controllers/TimeTrackerController.cs

```diff
@@ -20,14 +21,20 @@ public TimeTrackerController(ITimeTrackerData trackerData)
     [HttpPost("add-timeTracker")]
     public async Task<IActionResult> Add(TimeTrackerRequest entity)
     {
+        if (!TimeTrackerValidator.IsValid(entity))
+            return BadRequest("Invalid TimeTracker!");
         var timeTracker = _map.Map<TimeTracker>(entity);
+        if (!TimeTrackerValidator.IsValid(timeTracker))
```

Comment:
Why is the request tested twice? You should validate the configuration inside the validation helper: https://docs.automapper.org/en/stable/Configuration-validation.html

Reply:
I removed this line. The mapping is correct.

Reply:
Also, add the configuration validator for automaper.